packages/smooth_app/lib/pages/product/product_page/header/product_page_tabs.dart

```diff
@@ -189,6 +199,32 @@ class ProductPageTabBar extends StatelessWidget {
 
     return tabs;
   }
+
+  static Widget? _extractPrefix(Product product, KnowledgePanelTitle title) {
+    final String? attribute = switch (title.topics?.firstOrNull) {
+      'health' => Attribute.ATTRIBUTE_NUTRISCORE,
+      'environment' => Attribute.ATTRIBUTE_ECOSCORE,
+      _ => null,
+    };
+
+    if (attribute == null) {
+      return null;
+    }
+
+    final List<Attribute> attributes = getPopulatedAttributes(product, <String>[
+      attribute,
+    ], <String>[]);
+
+    if (attributes.isEmpty) {
+      return null;
+    }
+
+    final CardEvaluation eval = getCardEvaluationFromAttribute(
+      attributes.first,
```

Comment:
This is a methodology we need to let the server know about. We don't currently have anything server side @g123k ,  right ?